### PR TITLE
More directive language.

### PR DESCRIPTION
**Pull request by yhontyk**
> Read our [Contribution guide](https://github.com/eclipse/che-docs/blob/master/CONTRIBUTING.adoc) before submitting a PR.

### What does this PR do?
Introduces some style language improvements.

### CODE REVIEW

**Comment by yhontyk** [comment]
Blocked until style convections are discussed and agreed upon.

File: src/main/pages/che-7/end-user-guide/proc_overriding-devfile-values-using-factory-parameters.adoc [hunk 1]
```diff
@@ -72,14 +70,14 @@ projects:
 ...
 ----
 
-To add or override source `branch` value, the following factory URL can be used:
+To add or override source `branch` value, you can use the following factory URL:
```
Comment (themr0c) [inline src/main/pages/che-7/end-user-guide/proc_overriding-devfile-values-using-factory-parameters.adoc]:
```suggestion
To add or override source `branch` value, use the following factory URL:
```

File: src/main/pages/che-7/end-user-guide/proc_overriding-devfile-values-using-factory-parameters.adoc [hunk 2]
```diff
@@ -113,15 +111,15 @@ projects:
 ...
 ----
 
-To add or override `persistVolumes` attribute value, the following factory URL can be used:
+To add or override `persistVolumes` attribute value, you can use the following factory URL:
```
Comment (themr0c) [inline src/main/pages/che-7/end-user-guide/proc_overriding-devfile-values-using-factory-parameters.adoc]:
```suggestion
To add or override `persistVolumes` attribute value, use the following factory URL:
```

File: src/main/pages/che-7/end-user-guide/proc_overriding-devfile-values-using-factory-parameters.adoc [hunk 3]
```diff
@@ -33,14 +33,14 @@ projects:
 ...
 ----
 
-To add or override `generateName` value, the following factory URL can be used:
+To add or override `generateName` value, you can use the following factory URL:
```
Comment (themr0c) [inline src/main/pages/che-7/end-user-guide/proc_overriding-devfile-values-using-factory-parameters.adoc]:
```suggestion
To add or override `generateName` value, use the following factory URL:
```

**Review by themr0c — APPROVED** [review]

**Pull request merged**